tlatools/org.lamport.tlatools/src/tla2sany/semantic/ErrorCode.java

```diff
@@ -63,7 +63,7 @@ public enum ErrorCode {
    * never be reached since the semantic checking process will have failed
    * before those points.
    */
-  SUSPECTED_UNREACHABLE_CHECK (4004, ErrorLevel.ERROR),
+  SUSPECTED_UNREACHABLE_CHECK (4004, ErrorLevel.ERROR, -1),
```

Comment:
`-1` a negative number of parameters?

Reply:
Flag value for variable number of parameters.

Reply:
Consider turning it into a constant (`private static final int VAR_PARAMS = -1;`).